katib/katib-controller/base/katib-controller-rbac.yaml

```diff
@@ -54,13 +54,10 @@ rules:
   resources:
   - experiments
   - experiments/status
-  - experiments/finalizers
```

Comment:
Why are these removed?

Reply:
We don't have these roles in controller rbac: https://github.com/kubeflow/katib/blob/master/manifests/v1alpha3/katib-controller/rbac.yaml#L54. Do we need it?